content/millennium/r4/proprietary-codes.md

```diff
@@ -41,6 +41,18 @@ Example CodeableConcept displaying both standard and proprietary codes:
       }
     }
 
+## Financial
```

Comment:
move between Encounters and General Clinical

Reply:
https://github.com/cerner/fhir.cerner.com/pull/291/commits/1fd3fccdcbc14acbba047a902a80e96873ec8594#diff-f1a7812a3d6e9f4049fb6005eecfb1f5R61